frontend/language/src/nb.json

```
@@ -663,6 +663,12 @@
   "schema_editor.integer": "Heltall",
   "schema_editor.keyword": "Nøkkelord",
   "schema_editor.loading": "Laster",
+  "schema_editor.language": "Språk",
+  "schema_editor.language_add_language": "Legg til språk:",
```

The colon is missing in the English translation. You could add it in the English translation, or remove it in Norwegian and add it in the react component:

```suggestion
  "schema_editor.language_add_language": "Legg til språk",
```

---

fixed